File: 2-semestre/uri-online-judge/c/1142.c
#include <stdio.h>
 
int main() {
    int n, x, i;
    
    scanf("%d\n", &n);
    
    x = 1;
    
    for (i = 0; i < n; i++) {
        printf("%d %d %d PUM\n", x, x+1, x+2);

        x += 4;
    }
 
    return 0;
}

File: 2-semestre/uri-online-judge/javascript/1142.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let n = parseInt(lines.shift());
let x = 1;

for (let i = 0; i < n; i++) {
    console.log(`${x} ${x+1} ${x+2} PUM`);

    x += 4;
}
